inbm/dispatcher-agent/dispatcher/source/constants.py

```diff
@@ -26,22 +26,22 @@ class SourceParameters:
 
 @dataclass(kw_only=True, frozen=True)
 class ApplicationAddSourceParameters:
-    file_name: str
-    sources: list[str] = field(default_factory=lambda: [])
+    source_list_file_name: str
```

Comment:
Rename `file_name` to `source_list_file_name` in `ApplicationAddSourceParameters` to match the changes in other classes. [important]

Reply:
This is commenting on the code before the change.  Issue filed.